site/pages/api/hootsuite-callback.api.ts

```diff
@@ -101,10 +99,27 @@ const hootsuiteCallback = async (req: NextApiRequest, res: NextApiResponse) => {
 
         const userId: string = userDetails.data.id;
 
+        let refreshTokens = null;
+        try {
+            refreshTokens = (await getParametersByPath(`/social/${session.orgId}/hootsuite`)) ?? null;
+        } catch (e) {}
+
+        const refreshTokenExists = refreshTokens?.Parameters?.filter((rt) => rt.Name?.includes(userId));
+
+        if (refreshTokenExists && refreshTokenExists.length > 0) {
+            await Promise.all(
+                refreshTokenExists.map(async (refreshToken) =>
+                    refreshToken.Name ? await deleteParameter(refreshToken.Name) : "",
+                ),
+            );
+        }
+        await delay(3000);
```

Comment:
Can we add comment to indicate there is eventual consistency is SSM for deleteParameter api

Reply:
done